modules/benchmark/R/align_data.R

```diff
@@ -4,16 +4,14 @@
 ##' @param model.calc data.frame
 ##' @param obvs.calc data.frame
 ##' @param var data.frame
-##' @param start_year numeric
-##' @param end_year numeric
 ##' @return dat
 
 ##' @author Betsy Cowdery
 
 
 ## Align timeseries data using different functions
 
-align_data <- function(model.calc, obvs.calc, var, start_year, end_year, align_method = "match_timestep") {
+align_data <- function(model.calc, obvs.calc, var, align_method = "match_timestep") {
```

Comment:
this is going to break [PDA](https://github.com/PecanProject/pecan/blob/develop/modules/assim.batch/R/pda.get.model.output.R#L114), please grep where the function's being called through the code and change their arguments too when you make a change to a fcn arguments.